DigitalOffice/Tests/ProjectServiceTests/Mappers/ProjectMapperTests.cs

```diff
@@ -61,5 +101,41 @@ public void ReturnsProjectModelSuccessfully()
 
             SerializerAssert.AreEqual(expected, result);
         }
+
+        [Test]
+        public void SuccessfulNewRequestToDbMappingTest()
+        {
+            var newProject = newRequestToDbMapper.Map(newRequest);
+
+            var expectedDbProject = new DbProject
+            {
+                Id = newProject.Id,
+                Name = newProject.Name,
+                DepartmentId = newRequest.DepartmentId,
+                Description = newRequest.Description,
+                Deferred = false,
+                IsActive = newRequest.IsActive
+            };
+
```

Comment:
why is the expected result built from the result to be checked?

Reply:
Now THERE we have a problem that you mentioned in the section above. Right there we should not assign from newProject (That is to be checked) but it's not even my code lol

nevermind I'll fix this